dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/thread/ObservableBlockingQueue.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.core.internal.thread
+
+import com.datadog.android.internal.thread.NamedRunnable
+import java.util.concurrent.LinkedBlockingQueue
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.atomic.AtomicLong
+
+internal open class ObservableBlockingQueue<E : Any>(
```

Comment:
very minor suggestion

```suggestion
internal open class ObservableLinkedBlockingQueue<E : Any>(
```